backend/WSS_Site/settings.py

```diff
@@ -207,6 +207,15 @@
     },
 }
 
+PAYMENT_SETTING = {
+    "MERCHANT": '5ff4f360-c10a-11e9-af68-000c295eb8fc',
```

Comment:
Use env var. (like `SECRET_KEY`).  
And for local run you can specify that env var like: `MERCHANT_ENV_VAR=sss sudo docker-compose up` ...

Reply:
Please check out all environment variables in CD configs.